sox.dialog.js

```diff
@@ -141,21 +138,12 @@
 
           const $feature = $soxSettingsDialogFeatures.find('input#' + name).parent();
           $feature.append($settingsToggle);
-
-          if ($div.has('.sox-sprite-info').length) {
-            $info.after($settingsDiv);
-          } else {
-            $feature.append($settingsDiv);
-          }
+          $div.has('.sox-sprite-info').length ? $info.after($settingsDiv) : $feature.append($settingsDiv);
         }
       }
 
       // display sox version number in the dialog
-      if (version != 'unknown' && version !== null) {
-        $soxSettingsDialogVersion.text(' v' + (version ? version.toLowerCase() : ''));
-      } else {
-        $soxSettingsDialogVersion.text('');
-      }
+      version != 'unknown' && version !== null ? $soxSettingsDialogVersion.text(' v' + (version ? version.toLowerCase() : '')) : $soxSettingsDialogVersion.text('');
```

Comment:
Again, I think it's more readable as an `if` instead of ternary expression in this case, especially as the code for matching a version is a longer line.